### PR TITLE
Doc 6616 dt quick start

### PR DESCRIPTION
**Pull request by rhetoric101**
### Tell us why

We wanted to streamline the distributed tracing setup documentation with a quick-start guide.

### Anything else you'd like to share?

For details, see doc-6659.

### CODE REVIEW

File: src/content/docs/distributed-tracing/index.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    * [What is this advanced feature called Infinite Tracing?](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing/)
+  </LandingPageTile>
+ </LandingPageTileGrid>
+
+<ButtonLink
+  role="button"
+  to="/docs/distributed-tracing"
+  variant="normal"
+>
+  View main distributed tracing docs
```
Comment (zuluecho9) [inline src/content/docs/distributed-tracing/index.mdx:53]: this link, locally anyway, was just sending me to the same page (the landing page). NOt sure if we have to adjust that to show the actual docs and not the landing page; can't remember how we do that, but is probably present in other landing pages for the 'see all docs' link.

File: src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        [Detect anomalous spans](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data#anomalous-spans)
```
Comment (zuluecho9) [inline src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx:78]: for me locally, these links about UI were just taking me to the UI doc, without the anchor links working. Not sure why that'd be.

File: src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      </td>
+
+      <td>
+        You can filter charts using many data points, so you can analyze trace data in different ways.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [Customize queries and dashboards](/docs/insights/use-insights-ui/manage-dashboards/create-edit-copy-insights-dashboards)
```
Comment (zuluecho9) [inline src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx:108]: I'd prob link this to 'INtro to NRQL' doc instead. For getting started building custom queries, charts.

File: src/content/docs/distributed-tracing/enable-configure/quick-start.mdx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+import zipkinlogo from './images/zipkinlogo.png'
+
+import newreliclogo from './images/newreliclogo.png'
+
+To set up distributed tracing, you'll complete these three general steps: 
+
+  ![The three steps to set up distributed tracing](./images/three-steps.png "three-steps.png") 
+
+1. **Identify services:** Identify the endpoints, services, languages, and systems that are used to complete this request. Here's an example of how you might identify services in a request:
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/enable-configure/quick-start.mdx:43]: @zuluecho9, good point! I'll try to clarify the relationship between the image and the text!

File: src/content/docs/distributed-tracing/enable-configure/overview-enable-distributed-tracing.mdx [hunk 5]
```diff
@@ -91,6 +80,20 @@ We offer a range of agents to capture trace data from your applications:
...
+
+If you are collecting telemetry data with selected open source telemetry tools, you can send your data to New Relic with our integrations:
+
+* [OpenTelemetry](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-exporter)
+* [Istio](/docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-istio-integration)
+* [Kamon](/docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/kamon-reporter)
+* [OpenCensus](/docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-opencensus-integration)
+
+
+<Callout variant="tip">
```
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/enable-configure/overview-enable-distributed-tracing.mdx:93]: @zuluecho9 , I removed this callout for X-Ray and pushed it into the preceding bullet list.

File: src/content/docs/distributed-tracing/index.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/index.mdx:53]: @zuluecho9, good catch! I corrected this with this: `to="/docs/distributed-tracing/table-of-contents"`

File: src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx:78]: @zuluecho9, I corrected these!

File: src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx:108]: @zuluecho9, I switched this to NRQL.

**Comment by rhetoric101** [comment]
Reviewer: I made some last-minute changes to images. Could you confirm that images load correctly on the following pages:

* Distributed tracing landing page.
* Distributed tracing > Concepts > Introduction to distributed tracing
* Distributed tracing > Setup > Quick start (This page now has an arrow diagram that has a transparent background).

**Comment by bradleycamacho** [comment]
@rhetoric101 I might be missing something, but I don't see any of those paths in the navigation 🤔

**Comment by rhetoric101** [comment]
@bradleycamacho, I'm not sure why you're not seeing these. Here's what my left-navigation looks like (see attached). If you're not seeing these, then we have a problem.
left-nav-check

**Pull request merged**